.github/actions/bazel/action.yaml

```diff
@@ -73,46 +73,42 @@ runs:
           ${{ steps.metrics-tmpdir.outputs.dir }}/bazel-bep-*.pb.gpg
           ${{ steps.metrics-tmpdir.outputs.dir }}/profile-*.json
 
-
-      # Remove unnecessary entries from the JSON execlogs and create a CSV out of them
-    - name: Clean up execlogs
+    - name: Build execution_log_compact_to_csv
       shell: bash
       if: (success() || failure()) && inputs.execlogs-artifact-name != ''
+      env:
+        # Used by the bazel wrapper
+        BUILDBUDDY_LINKS: '[CI Job](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})'
       run: |
-        execlogs_json_in='${{ steps.metrics-tmpdir.outputs.dir }}/execlogs.json'
-        find '${{ steps.metrics-tmpdir.outputs.dir }}' -name 'execlog-*.json' \
-          -exec cat {} \; > "$execlogs_json_in"
-
-        execlogs_csv_out='${{ steps.metrics-tmpdir.outputs.dir }}/execlogs.csv'
-        if ! [ -s "$execlogs_json_in" ]; then
-          echo "no execlogs found"
-          exit 0
+        set -euo pipefail
+        if [ -x bazel-bin/bazel/execution_log_compact_to_csv ]; then
+          echo "bazel-bin/bazel/execution_log_compact_to_csv already exists, skipping build"
+        else
+          # Here we overwrite the PATH with our custom bazel wrapper to ensure
```

Comment:
is there a reason for manually setting the path instead of using the action?

Reply:
I copy pasted this from the "Run bazel commands" step above. Don't we need to set the PATH such that we use our own `bazel` wrapper?

Reply:
oh sorry, I didn't realize this was inside the bazel action itself; my bad!